iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/BusbarSectionPosition.java

```diff
@@ -21,10 +21,16 @@ default String getName() {
         return NAME;
     }
 
+    /**
+     * The position of the busbar relative to other busbars.
```

Comment:
What is a bus bar definition here ? We only have `busBarSection` objects in IIDM. Can you be more precise, maybe adding vertical or horizontal positions and the term "in general" for the interpretation if you prefer.

Reply:
it is busbar section

Reply:
We should not speak about vertical or horizontal, it is just an interpretation of SLD drawing.